docs/src/codegen.md

```diff
@@ -42,8 +42,9 @@ Once you are done recording click the **cancel** button or close the browser win
 
 ### Record at Cursor
 
-To record from a specific point in your test move your cursor to where you want to record more actions and then click the **Record at cursor** button from the Testing sidebar. If your browser window is not already open then first run the test with 'Show browser' checked and then click the **Record at cursor** button.
+To record from a specific point in your test move your cursor to where you want to record more actions and then click the **Record at cursor** button from the Testing sidebar.
 
+**Important:** This feature does not start the test and execute the lines before the cursor. It only attaches to an existing VSCode web view and gets it into recording mode. You have to get VSCode web view into a correct state manually. The easiest way to do this is by adding a breakpoint where you want to continue recording, starting the test in debugging mode and exiting the debugger after it stops at your breakpoint. You have to have 'Show browser' checked in the sidebar in order for VSCode web view to remain open after exiting the debugger.
```

Comment:
I guess by "VSCode web view" you mean the browser?

> The easiest way to do this is by adding a breakpoint where you want to continue recording, starting the test in debugging mode and exiting the debugger after it stops at your breakpoint.

This is misleading, we cannot record actions under the debugger and after resuming the execution playwright will continue to run the test till the end.

I don't think the new explanation is clearer then the existing one which describes two exact steps that one needs to perform.

Reply:
> I guess by "VSCode web view" you mean the browser?

Kind of. I think VS code web view is a more accurate accurate term, because the browser that one launches manually wouldn't work. It has vs code icon in the OS taskbar so it seems it's a vs code window that has a browser embedded. I'm fine adjusting that. I could call it "vs code lauched browser" to distinguish from user launched instances.

> This is misleading, we cannot record actions under the debugger and after resuming the execution playwright will continue to run the test till the end.

I'll adjust instructions so it is clear that I'm not suggesting to debug and record the test at the same time. What I was trying to say is that **before** starting to record the test - you put the breakpoint, debug the test and when it breaks at that line, you stop the debugger - **not** resume execution. It does not continue runing the test after the debugger is stopped. When the debugger is stopped, you would start recording the test at cursor.